epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -94,6 +94,11 @@ String getterCode() {
     return isPrivate ? getterMethodName + "()" : name;
   }
 
+  // Special case to avoid generating recursive getter if field and its getter names are the same
+  String superGetterCode() {
```

Comment:
Yeah, I don't like this solution either, but I couldn't find anything better.
I tried to handle it in the more general way same as in `setterCode()`, but failed because getter is used in `equals` and it messes it up.